app/Http/Controllers/Frontend/ChangelogController.php

```diff
@@ -19,6 +20,11 @@ public function renderChangelog(): View {
             return json_decode($json, true);
         });
 
+        // prevent changelog from showing future versions
+        $currentVersion = VersionController::getVersion();
+        if (str_starts_with($currentVersion, 'v') && $changelog['entry'][0]['title'] !== $currentVersion) {
+            array_shift($changelog['entry']);
+        }
```

Comment:
Bin mir unsicher, ob ich das richtig verstehe... was genau ist hier dein Ziel?
Lokal wird mir die "unreleaste" Version weiterhin angezeigt, was nach dem Titel des PRs ja eher nicht sein sollte?

![image](https://github.com/Traewelling/traewelling/assets/4103693/7ce88595-fe38-4b72-8e38-df825aab76c9)

Reply:
Das funktioniert nur in Kombination mit einer ausgefüllten VERSION-Datei. `VersionController::getVersion()` prüft erst, ob VERSION vorhanden ist und falls nicht, wird auf GIT geschaut. Mit #1926 zusammen sollte das eigentlich perfekt funktionieren.